src/transformers/models/beit/modeling_beit.py

```diff
@@ -86,6 +87,31 @@ def extra_repr(self) -> str:
         return "p={}".format(self.drop_prob)
 
 
+@dataclass
+class BeitModelOutputWithPooling(BaseModelOutputWithPooling):
```

Comment:
This should be added to the BeiT model doc page under a new section called "BeiT specific outputs". See the XLNet model doc source page for an example.

Reply:
I looked at this for reference : https://github.com/huggingface/transformers/blob/master/docs/source/model_doc/xlnet.rst#xlnet-specific-outputs

and added the `Specific Output` section.